### ISSUE TITLE
Vectorized subtype comparison

### ISSUE THREAD

**Issue by CameronBieganek** [JuliaLang/julia#34839]
Do we expect the following syntax to work?

```julia
julia> [Int, Float64, String, Bool] .<: Union{Int, String}
ERROR: syntax: space not allowed after ":" used for quoting
```

It works if you call `<:` as a broadcasted function:

```julia
julia> (<:).([Int, Float64, String, Bool], Union{Int, String})
4-element BitArray{1}:
 1
 0
 1
 0
```

Version info:

```julia
Julia Version 1.3.1
Commit 2d5741174c (2019-12-30 21:36 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.6.0)
  CPU: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_EDITOR = vim
```

**Comment by JeffBezanson** [comment]
This certainly doesn't come up very often, but I guess we could make it work. Just need to add the operators to the `add-dots` list.

**Comment by tyler2181** [comment]
Excuse me

**Comment by JeffBezanson** [comment]
Fixed by #35085
